charts/camunda-platform/templates/_helpers.tpl

```diff
@@ -105,3 +104,7 @@ Subcharts can not access values from other sub-charts or the parent, global only
         http://{{ include "common.names.dependency.fullname" (dict "chartName" "keycloak" "chartValues" . "context" $) | trunc 20 | trimSuffix "-" }}:80{{- include "identity.keycloak.contextPath" . -}}{{ $keycloakRealmPath }}
     {{- end -}}
 {{- end -}}
+
+{{- define "camundaPlatform.identityBackendUrl" -}}
+http://{{ include "common.names.dependency.fullname" (dict "chartName" "identity" "chartValues" . "context" $) }}
```

Comment:
I'm not sure if there's maybe a better way to get the fullname from the identity subchart?
I wanted to use `{{ include "identity.fullname" . }}` which doesn't work though when this template is used in the Web Modeler context.
Also, `identity.service.port` would actually need to be included in the URL (in case someone chose a different value than the default `80`).

Reply:
@jfriedenstab this is one of the main obstacles we have now.
By design, Helm treats each sub-chart as an independent chart, i.e. sub-charts cannot access each other.

But anyway, what do you want to access? Did you take a look at Operte/Tasklist, for example?
The common keys are access under `.global.identity`.

Reply:
@aabouzaid: I'd like to set up a backchannel communication between Web Modeler and Identity (because the Web Modeler calls the Identity API internally). That's why I need the Identity service name and port.

Reply:
Then I guess you have to define a global value